applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -315,7 +315,7 @@ upload_csv(Context) ->
     Now = erlang:now(),
     {'ok', {Count, Rates}} = process_upload_file(Context),
     lager:debug("trying to save ~b rates (took ~b ms to process)", [Count, wh_util:elapsed_ms(Now)]),
-    _  = crossbar_doc:save(cb_context:set_doc(Context, Rates), [{'publish_doc', 'false'}]),
+    lists:foreach(fun(Rate) -> crossbar_doc:ensure_saved(cb_context:set_doc(Context, Rate), [{'publish_doc', 'false'}]) end, Rates),
```

Comment:
and did you see @jamesaimonetti comment ? i really hate to use `ensure_saved` unless there's a good reason for that. one example is saving account doc in account db and accounts db, ie, saving the same document in two different locations.